docs/en/05_Releases_and_changelogs/cwp_2.4.0.md

```diff
@@ -0,0 +1,44 @@
+# 2.4.0
```

Comment:
We should create this in the 2.4 branch. There's also a changelog template which Cow would use to create this file if it had done during a release process - you could use that as a base for this file. See https://github.com/silverstripe/cwp-recipe-kitchen-sink/blob/master/.changelog.md.twig

Reply:
I've retargeted to 2.4, and used the "New features" header from the template. I'll leave the other templated bits to the CWP release managers, probably safer than copy pasting placeholders which then get missed later during the actual release.